tapioca.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |spec|
   spec.metadata["allowed_push_host"] = "https://rubygems.org"
 
   spec.add_dependency("bundler", ">= 2.2.25")
+  spec.add_dependency("logger")
```

Comment:
Where does Tapioca have a dependency on the `logger` gem? I am not sure if this is correct.

Reply:
This has since been resolved. Yard has dropped the logger dependency in https://github.com/lsegal/yard/commit/a589b9a9af04398b977e328bbd37694672dd3ef7 and released [0.9.37](https://github.com/lsegal/yard/releases/tag/v0.9.37)